### ISSUE TITLE
Confusing help/debug message for payment_intents/confirm

### ISSUE THREAD

**Issue by enugent-stripe** [stripe/stripe-cli#372]
### Issue
I was trying to confirm a Payment Intent. My command was wrong, but the error message I got didn't really help me get back on track:
```
> stripe payment_intents confirm -d payment_method=pm_some_token -d return_url="https://example.com"
confirm only takes 1 argument. See `stripe confirm --help` for supported flags and usage
```
"Only takes 1 argument" suggests to me that one of those two `-d` flagged options shouldn't be there. In reality, it's complaining that I haven't added the Payment Intent token like so:
```
stripe payment_intents confirm pi_some_token
```

Moreover, `stripe confirm --help` doesn't work:
```
> stripe confirm --help
Unknown command "confirm" for "stripe". Did you mean "config"?
If not, see "stripe --help" for a list of available commands.
```
### Expected Behavior
I'd expect `stripe confirm --help` to give a standard help/man type page. The generic error message for confirming the PI itself could also be improved.

### Steps to reproduce
Rerun the commands above on `stripe version 1.3.2`.

### Traceback
N/A

### Environment
macOS, though not applicable

**Comment by aragalie** [comment]
+1 and i think the correct error message should be `stripe payment_intents confirm --help` (which works)

**Comment by richardm-stripe** [comment]
(Addressed in v1.4.3)
